robo_soccer_platform/lib/robo_soccer_platform_web/live/player.ex

```diff
@@ -34,7 +34,6 @@ defmodule RoboSoccerPlatformWeb.Player do
             name="room_code"
             value={@room_code}
             label="Kod Pokoju"
-            errors={@form["errors"]}
```

Comment:
why?

Reply:
because form["errors"] is only updated when an invalid username is provided, causing errors about the username to be displayed below both fields